### ISSUE TITLE
pnpm add -D @quasar/vite-plugin sass@1.32.12 

### ISSUE THREAD

**Issue by yangzxzx** [quasarframework/quasar#13955]
### What happened?

unmet peer vite@^2.0.0: found 3.0.0
![image](https://user-images.githubusercontent.com/47587488/178952653-afdf1711-9b4c-420b-bb34-e9e14eb74118.png)

### What did you expect to happen?

done

### Reproduction URL

https://quasar.dev/start/vite-plugin

### How to reproduce?

pnpm add -D @quasar/vite-plugin sass@1.32.12

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite)

**Comment by lhoeg** [comment]
Vite 3.0.0 was released yesterday but vite-plugin/package.json requires vite "^2.0.0" under peerDependencies.

Should be solved by relaxing the requirement to ">2.0.0", please;

 "peerDependencies": {
     "quasar": "^2.0.0",
-     "vite": "^2.0.0",
+    "vite": ">=2.0.0",
     "vue": "^3.0.0"
   },

**Comment by alex-unruh** [comment]
I needed to change this step described in documentation:
```
npm init vite@latest my-vue-app -- --template vue
```
to this one... 
```
npm init vite@2.9 my-vue-app -- --template vue
```
